configs/projects/_shared-pe-bolt-server.rb

```diff
@@ -18,7 +18,7 @@
   exit(1)
 end
 
-if platform.name =~ /^redhatfips-7-.*/
+if platform.name =~ /^redhatfips-.*/
```

Comment:
@nmburgan @actowery @barriserloth do you know if these system_openssl changes are still needed? This logic seems to have been added for most projects in https://github.com/puppetlabs/puppet-runtime/pull/228 and https://github.com/puppetlabs/puppet-runtime/pull/214.

For more context, we switched to building openssl 1.1.1 on redhatfips-7 where the system openssl is 1.0.2, and on redhatfips-8 where the system openssl is also 1.1.1. I'm not sure what the situation was back in 2019 when these changes were added.